react/features/polls/components/PollChoice.js

```diff
@@ -0,0 +1,188 @@
+// @flow
+
+import React, { Component } from 'react';
+import { FieldTextStateless } from '@atlaskit/field-text';
+
+type Props = {
+
+    /**
+     * Item ID in list.
+     */
+    id: string,
+
+    /**
+     * Container delete method.
+     */
+    deleteHandler?: Function,
```

Comment:
Generally we name methods like this something like onDelete or onDeleteCallback, not handler.